docs/src/edmf_equations.md

```diff
@@ -0,0 +1,230 @@
+  K^j = \tfrac{1}{2} (\boldsymbol{u}_{h}^j \cdot \boldsymbol{u}_{h}^j + 2 \boldsymbol{u}_{h}^j \cdot I_{c} (\boldsymbol{u}_{v}^j) + I_{c}(\boldsymbol{u}_{v}^j \cdot \boldsymbol{u}_{v}^j)),
+  ```
+  where ``\boldsymbol{u}_{h}^j`` is defined on cell-centers, ``\boldsymbol{u}_{v}^j`` is defined on cell-faces, and ``I_{c} (\boldsymbol{u}_{v})`` is interpolated using covariant components.
+
+* No-flux boundary conditions are enforced by requiring the third contravariant component of the face-valued velocity at the boundary, ``\boldsymbol{\tilde{u}}^{v,j}``, to be zero. The vertical covariant velocity component is computed as
+  ```math
+  \tilde{u}_{v}^j = \tfrac{-(u_{1}g^{31} + u_{2}g^{32})}{g^{33}}.
+  ```
+!!! todo
+    Is the boundary condition for ``\boldsymbol{\tilde{u}}^{v,j}`` correct???
+
+## Equations and discretizations
+
+### Mass
+
+Follows the continuity equation
+```math
+\frac{\partial}{\partial t} \hat{\rho}^j = - \nabla \cdot (\hat{\rho}^j \boldsymbol{u}^j)  + RHS
+```
+
+This is discretized using the following
+```math
+\frac{\partial}{\partial t} \hat{\rho}^j
+= - D_h[ \hat{\rho}^j (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v^j))] - D^c_v \left[WI^f( J, \hat{\rho}^j) \tilde{\boldsymbol{u}^j} \right] + RHS
+```
+
+!!! todo
+    Is the weighting factor ``\hat{\rho}^j``???
+
+### Momentum
+
+Uses the advective form equation
+```math
+\frac{\partial}{\partial t} \boldsymbol{u}^j  = - (2 \boldsymbol{\Omega} + \nabla \times \boldsymbol{u}^j) \times \boldsymbol{u}^j - \frac{1}{\rho^j} \nabla (p - p_{\text{ref}})  + \boldsymbol{b}^j - \nabla K^j + RHS
+```
+
+!!! todo
+    Why are we not dividing by ``\hat{\rho}^j`` to go from flux form to advective form???
+
+#### Horizontal momentum
+
+!!! todo
+    Should we remove the horizontal momentum equation ???
+
+By breaking the curl and cross product terms into horizontal and vertical contributions, and removing zero terms (e.g. ``\nabla_v  \times \boldsymbol{u}_v = 0``), we obtain
+
+```math
+\frac{\partial}{\partial t} \boldsymbol{u}_h  =
+  - (\nabla_v \times \boldsymbol{u}_h +  \nabla_h \times \boldsymbol{u}_v^j) \times \boldsymbol{u}^{v,j}
+  - (2 \boldsymbol{\Omega}^v + \nabla_h \times \boldsymbol{u}_h) \times \boldsymbol{u}^h
+  - \frac{1}{\rho^j} \nabla_h (p - p_{\text{ref}})  - \nabla_h (\Phi + K^j) ,
+```
+where ``\boldsymbol{u}^h`` and ``\boldsymbol{u}^{v,j}`` are the horizontal and vertical _contravariant_ vectors from sub-domain. The effect of topography is accounted for through the computation of the contravariant velocity components (projections from the covariant velocity representation) prior to computing the cross-product contributions.
+
+The ``(\nabla_v \times \boldsymbol{u}_h + \nabla_h \times \boldsymbol{u}_v^j) \times \boldsymbol{u}^{v,j}`` term is discretized as:
+```math
+\frac{I^c((C^f_v[\boldsymbol{u}_h] + C_h[\boldsymbol{u}_v^j]) \times (I^f(\hat{\rho}^j J)\tilde{\boldsymbol{u}}^v,j))}{\hat{\rho}^j J}
+```
+!!! todo
+    Is the weighting factor ``\hat{\rho}^j J``???
```

Comment:
The correct weighting factor would be ``\hat{\rho}^j J``. However, this can go to zero, creating difficulties. The safer implementation route would be to use ``\rho^j J`` first but flag it to be changed later once this is working. Otherwise, we need to build in regularization in the division by zero case (add +eps in denominator).